scripts/cncli-leaderlog.sh

```diff
@@ -62,8 +62,8 @@ calculateLeaderLog ()
 
         if [[ $binCardanoCliMajorVersion -eq 1 ]];
         then
-            poolTotalStake=$(echo "$poolSnapshot" | jq -r .poolStakeMark)
-            poolActiveStake=$(echo "$poolSnapshot" | jq -r .activeStakeMark)
+            poolTotalStake=$(echo "$poolSnapshot" | jq -r .pool${3^})
```

Comment:
`jq` should not be used as it has a bug parsing large numbers. look at the `next` script for how it parses this data.

Reply:
Now using python instead of jq to process the json. Not using the regex in favor of readability and maintainability.

Reply:
Please just use grep and cut instead of introducting a python dependency. See how it's done here:

https://github.com/cardano-community/cncli/blob/develop/scripts/cncli-leaderlog-next.sh#L8-L15

Reply:
Done